qutebrowser/utils/jinja.py

```diff
@@ -127,3 +130,33 @@ def render(template, **kwargs):
 
 environment = Environment()
 js_environment = jinja2.Environment(loader=Loader('javascript'))
+
+
+@functools.lru_cache()
+def template_config_variables(template: str) -> typing.FrozenSet[str]:
+    """Return a frozenset of config variables that is used in template."""
+    pending = [environment.parse(template)]  # a list of unvisited nodes
```

Comment:
I don't think the comment adds much - you could even call the variable `unvisited_nodes` instead (but `pending` is fine too, IMHO).